python/pyarrow/tests/test_parquet.py

```diff
@@ -577,45 +582,64 @@ def test_parquet_metadata_api():
 
 @parquet
 @pytest.mark.parametrize(
-    'data, dtype, min_value, max_value, null_count, num_values',
+    (
+        'data',
+        'type',
+        'physical_type',
+        'min_value',
+        'max_value',
+        'null_count',
+        'num_values',
+        'distinct_count'
```

Comment:
I'm curious, why is `distinct_count` always None in the examples below?

Reply:
Uwe said:

> distinct_count was not computed, so I would expect np.nan or None as the result.

Reply:
I see, can you add a small comment?

Reply:
Will do, just doing some packaging stuff.